packages/client/src/editor/components/StickyNote.tsx

```diff
@@ -1,17 +1,38 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState, useRef } from 'react';
 import EditIcon from '@material-ui/icons/Edit';
 import DeleteIcon from '@material-ui/icons/Delete';
 import { UUID, Note } from '../../types';
 
 interface NoteProps {
   save: (id: UUID, content: string) => void;
   delete: (id: UUID) => void;
+  move: (id: UUID, x: number, y: number) => void;
   note: Note;
 }
 
 const StickyNote: React.FunctionComponent<NoteProps> = props => {
   const { note } = props;
 
+  const [mouseDown, setMouseDown] = useState(false);
+  const [pos, setPos] = useState({x: note.position.x, y: note.position.y});
+  const ref = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (ref.current) {
```

Comment:
I think the case in which using ref.current _could_ provide benefits if the style update happened directly in onMouseMove handler -- then it would happen earlier than the react rerender. But I tried doing that, and it still didn't guarantee mouse not leaving the note, so I don't think it's worth the extra complexity.